src/main/java/org/craftercms/deployer/impl/lifecycle/aws/DuplicateS3LifecycleHook.java

```diff
@@ -144,11 +139,11 @@ protected void doExecute(Target target) throws DeployerException {
             return;
         }
         List<String> paths = getItemPathList(repoPath);
-        AmazonS3 client = buildClient(builderConfigurer);
+        S3AsyncClient client = buildClient(builderConfigurer);
         try {
-            AwsUtils.copyObjects(client, threadPoolTaskExecutor::getThreadPoolExecutor, getBucket(srcS3Url, sourceSiteName), getS3BaseKey(srcS3Url, sourceSiteName),
+            AwsUtils.copyObjects(client, threadPoolTaskExecutor.getThreadPoolExecutor(), getBucket(srcS3Url, sourceSiteName), getS3BaseKey(srcS3Url, sourceSiteName),
                     getBucket(s3Url, siteName), getS3BaseKey(s3Url, siteName), paths);
-        } catch (InterruptedException e) {
+        } catch (Exception e) {
             throw new DeployerException(format("Interrupted while waiting for S3 content duplication from site '%s' to '%s'", sourceSiteName, siteName), e);
```

Comment:
Please adjust this message. It is not necessarily an interruption

Reply:
Updated